appium/api-mocks/apis/google/config.ts

```diff
@@ -0,0 +1,44 @@
+import { KeyUtil, KeyInfoWithIdentityAndOptionalPp } from '../../../api-mocks/core/crypto/key';
+
+export type TestKeyInfo = {
+  title: string, passphrase: string, armored: string | null, longid: string | null
```

Comment:
Maybe better if we change this to interface like below
```
export interface TestKeyInfo {
  title: string;
  passphrase: string;
  armored: string | null;
  longid: string | null;
}
```

Reply:
Could take a look at what we use in browser extension (that you recently worked on) and use the same format as there, so that key samples can be directly copied over.